adding new updates to Getting Started (#22250)
microsoftgraph/microsoft-graph-docs: includes/data-connect-quickstart-01.md

Additionally, you can use Data Connect to enable machine learning scenarios for your organization. In these scenarios, you can create applications that provide valuable information to your stakeholders, train machine learning models, and even perform forecasting based on large amounts of acquired data.

## Get started

In this tutorial, you'll create your first single-tenant Data Connect application. The following is a general flow that explains the Data Connect onboarding.

> ![A screenshot that explains the onboarding flow.](../concepts/images/data-connect-overview-flowchart.png)

Reviewer: Can we update this to the latest image in Michael's PR?

Author: @sannaga7  can you send me the image please?

5. **Developer runs their pipelines:** After the administrator has consented to the application, the developer might run their pipelines without any stalling for runtime consent. The pipeline creation and run via Azure Data Factory or Azure Synapse remains the same.  

> [!NOTE]

> > For existing tenants who already have Microsoft Graph Data Connect enabled and want to use this tutorial: notify your global administrator toggle Microsoft Graph Data Connect off and then on again in the [admin portal](https://admin.microsoft.com/adminportal/home#/Settings/Services/:/Settings/L1/O365DataPlan) to be able to follow this tutorial flow. For new customers who have not yet toggled Microsoft Graph Data Connect, please follow the steps in this tutorial below. No further action is needed on your part.

Reviewer 2: This note text is currently unclear. The following is a suggested revision:

If you have an existing tenant that already has Microsoft Graph Data Connect enabled, before you complete this tutorial, ask your global administrator to toggle Microsoft Graph Data Connect off and then on again in the [admin portal](https://admin.microsoft.com/adminportal/home#/Settings/Services/:/Settings/L1/O365DataPlan). If you're working in a new tenant that doesn't have Data Connect enabled, you don't need to take any action.

> > For existing tenants who already have Microsoft Graph Data Connect enabled and want to use this tutorial: please notify your global adminstrator toggle Microsoft Graph Data Connect off and then on again in the [admin portal](https://admin.microsoft.com/adminportal/home#/Settings/Services/:/Settings/L1/O365DataPlan) to be able to follow this tutorial flow. For new customers who have not yet toggled Microsoft Graph Data Connect on, please follow the steps in this tutorial below. No extra action is required from you.

Reviewer 3: ```suggestion
> For existing tenants who already have Microsoft Graph Data Connect enabled and want to use this tutorial: please notify your global administrator toggle Microsoft Graph Data Connect off and then on again in the [admin portal](https://admin.microsoft.com/adminportal/home#/Settings/Services/:/Settings/L1/O365DataPlan) to be able to follow this tutorial flow. For new customers who have not yet toggled Microsoft Graph Data Connect on, please follow the steps in this tutorial below. No extra action is required from you.
```

> > For existing tenants who already have Microsoft Graph Data Connect enabled and want to use this tutorial: please notify your global administrator toggle Microsoft Graph Data Connect off and then on again in the [admin portal](https://admin.microsoft.com/adminportal/home#/Settings/Services/:/Settings/L1/O365DataPlan) to be able to follow this tutorial flow. For new customers who have not yet toggled Microsoft Graph Data Connect on, please follow the steps in this tutorial below. No extra action is required from you.

Reviewer 3: ```suggestion
> For existing tenants who already have Microsoft Graph Data Connect enabled and want to use this tutorial: please notify your global administrator toggle Microsoft Graph Data Connect off and then on again in the [admin portal](https://admin.microsoft.com/adminportal/home#/Settings/Services/:/Settings/L1/O365DataPlan) to be able to follow this tutorial flow. For new customers who have not yet toggled Microsoft Graph Data Connect, please follow the steps in this tutorial below. No further action is needed on your part.

> > For existing tenants who already have Microsoft Graph Data Connect enabled and want to use this tutorial: please notify your global administrator toggle Microsoft Graph Data Connect off and then on again in the [admin portal](https://admin.microsoft.com/adminportal/home#/Settings/Services/:/Settings/L1/O365DataPlan) to be able to follow this tutorial flow. For new customers who have not yet toggled Microsoft Graph Data Connect, please follow the steps in this tutorial below. No further action is needed on your part.

Reviewer 4: ```suggestion
> For existing tenants who already have Microsoft Graph Data Connect enabled and want to use this tutorial: notify your global administrator toggle Microsoft Graph Data Connect off and then on again in the [admin portal](https://admin.microsoft.com/adminportal/home#/Settings/Services/:/Settings/L1/O365DataPlan) to be able to follow this tutorial flow. For new customers who have not yet toggled Microsoft Graph Data Connect, please follow the steps in this tutorial below. No further action is needed on your part.
```

> [!NOTE]

> > For reference, we have left guidance for our former consent process involving Privileged Access Management (PAM) in sections of our tutorial as tabs. We highly recommend following our new consent flow for a more simplified onboarding process.

Reviewer 3: ```suggestion
>We have included the guidance for our former consent process involving Privileged Access Management (PAM) in sections of our tutorial as tabs for reference purposes. However, we highly recommend following our new consent flow for a more simplified onboarding process.
```

> > We have included the guidance for our former consent process involving Privileged Access Management (PAM) in sections of our tutorial as tabs for reference purposes. However, we highly recommend following our new consent flow for a more simplified onboarding process.

Reviewer 2: Revise as:

The guidance for using the former consent process involving Privileged Access Management (PAM) is included in a separate tab for reference purposes. We recommend that you follow the new consent flow for a simplified onboarding process.

Also, you have two notes next to each other here, which can dilute the emphasis. See if you can rearrange the text to avoid having the two notes together.

> >We have included the guidance for our former consent process involving Privileged Access Management (PAM) in sections of our tutorial as tabs for reference purposes. However, we highly recommend following our new consent flow for a more simplified onboarding process.

Reviewer 3: ```suggestion
> We have included the guidance for our former consent process involving Privileged Access Management (PAM) in sections of our tutorial as tabs for reference purposes. However, we highly recommend following our new consent flow for a more simplified onboarding process.
```

         - An "admin" account that can access the [Microsoft 365 admin center](https://admin.microsoft.com/), specifically to approve the test application.  
   - Note the Microsoft 365 region where your tenant is located. If you create a new tenant, ensure it is one of the [regions supported by Microsoft Graph Data Connect](/graph/data-connect-datasets#regions).

   > [!NOTE] 

>    > You can't approve your own test application using the same account. Ensure that you have another member (or account) in your tenant that acts as an "admin".

Reviewer 2: Remove the quotes from "admin".

2. **Microsoft Azure subscription** 

   - If you don't have a Microsoft Azure subscription, visit the following link to get one (for free): https://azure.microsoft.com/free/.
   - Your Azure subscription must be in the same tenant as your Microsoft 365 tenant and both must be in the same Azure AD tenancy.
   - If your Azure subscription isn't in the same tenant as your Microsoft 365 tenant, you can associate your subscription with Azure AD in your Microsoft 365 tenant by following the steps listed in [Associate or add an Azure subscription to your Azure Active Directory tenant](/azure/active-directory/fundamentals/active-directory-how-subscriptions-associated-directory).

   > [!NOTE] 

>    > The screenshots and examples used in this tutorial are from a Microsoft 365 developer tenant with sample email from test users. You can use your own Microsoft 365 tenant to perform the same steps. No data is written to Microsoft 365. A copy of email data is extracted from all users in a Microsoft 365 tenant and copied to an Azure Blob Storage account. You maintain control over who has access to the data within the Azure Blob Storage.

Reviewer 3: ```suggestion
   > The screenshots and examples used in this tutorial are from a Microsoft 365 developer tenant with a sample email from test users. You can use your own Microsoft 365 tenant to perform the same steps. No data is written to Microsoft 365. A copy of email data is extracted from all users in a Microsoft 365 tenant and copied to an Azure Blob Storage account. You maintain control over who has access to the data within the Azure Blob Storage.
```
